musl doesn't support strtoll_l(), so replace it with a simple strtoll() call.

diff -Nru a/libcxx/include/locale b/libcxx/include/locale
--- a/libcxx/include/locale	2017-01-04 23:56:00.000000000 +0000
+++ b/libcxx/include/locale	2017-05-14 23:43:07.867204323 +0000
@@ -695,7 +695,7 @@
         typename remove_reference<decltype(errno)>::type __save_errno = errno;
         errno = 0;
         char *__p2;
-        long long __ll = strtoll_l(__a, &__p2, __base, _LIBCPP_GET_C_LOCALE);
+        long long __ll = strtoll(__a, &__p2, __base);
         typename remove_reference<decltype(errno)>::type __current_errno = errno;
         if (__current_errno == 0)
             errno = __save_errno;
@@ -735,7 +735,7 @@
         typename remove_reference<decltype(errno)>::type __save_errno = errno;
         errno = 0;
         char *__p2;
-        unsigned long long __ll = strtoull_l(__a, &__p2, __base, _LIBCPP_GET_C_LOCALE);
+        unsigned long long __ll = strtoull(__a, &__p2, __base);
         typename remove_reference<decltype(errno)>::type __current_errno = errno;
         if (__current_errno == 0)
             errno = __save_errno;
